stdlib/src/builtin/floatable.mojo

```diff
@@ -70,8 +70,8 @@ trait FloatableRaising:
 
     ```mojo
     @value
-    struct MaybeFloat(FloatableRasing):
-        var value: Variant[Float64, NoneType]
+    struct MaybeFloat(FloatableRaising):
+        var value: stdlib.utils.Variant[Float64, NoneType]
```

Comment:
Can you just say

```
from utils import Variant
```
after the mojo doc block and use `Variant` unqualified here, please?

Reply:
Sure, I'm gonna look at more examples, so I'll do that going forward.

Reply:
That'd be great, thanks! I think of examples as "what would we like users to write/model code after?"  In that lens, we'd want them to import said type if it's not already imported in the prelude, and then use it unqualified.